projects/cds/src/services/cds-endpoints.service.ts

```diff
@@ -0,0 +1,55 @@
+import { Injectable } from '@angular/core';
+import { DynamicTemplate } from '../../../core/src/config/utils/dynamic-template';
```

Comment:
`DynamicTemplate` is not currently exported by core. Would you prefer we copy `DynamicTemplate` and it's test into our project or export DynamicTemplate from core?

Reply:
:/
Do you expect that you will add more endpoints to the config in the future? 
I guess I'm just looking to simplify things and maybe remove this class altogether. 
Will @corey-aloia need to add some endpoints to justify having this complex logic for more than one URL?

Reply:
Or, maybe you can inherit our class and override some method? Or maybe use composition.

Reply:
Another option that we can consider is that we create a generic service that you can re-use. But this will require some time to prioritize. 
Until then, we can make this service "private" and not export it to clients. Then we can go back to it and adjust it to our liking.

Reply:
I've created our own copy of the DynamicTemplate class as this class wasn't exported from the @spartacus/core, so we couldn't re-use it anyway.